mysql/mysql.go

```diff
@@ -141,6 +141,8 @@ type Config struct {
 	Encoder session.Encoder
 	// Decoder is the decoder to decode session data. Default is session.GobDecoder.
 	Decoder session.Decoder
+	// InitTable is a flag to determine whether create a default session table automatically.
```

Comment:
```suggestion
	// InitTable indicates whether to create a default session table when not exists automatically.
```

Same for the other place.